src/database/models/trip.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Trip extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Trip.belongsTo(models.TripRequest, {
+        as: 'trip',
+        foreignKey: 'requestId'
+      });
+    }
+  }
+  Trip.init(
+    {
+      requestId: {
+        type: DataTypes.STRING,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE'
+      },
+      accomodationId: DataTypes.STRING,
+      departFrom: DataTypes.STRING,
+      departTo: DataTypes.STRING,
+      departDate: DataTypes.STRING,
```

Comment:
Check with @elvisiraguha . departTo is a location, thus should be an int

Reply:
Okay, thanks for the heads up @kipropbrian